src/utils/openmc/universe.py

```diff
@@ -1122,54 +1220,165 @@ def create_xml_subelement(self, xml_element):
             dimension.text = '{0} {1}'.format(self._center[0], \
                                               self._center[1])
 
-        # FIXME: The following must be revised for hexagonal lattice ordering
-        # Export the Lattice nested Universe IDs - column major for Fortran
-        universe_ids = '\n'
+        # Export the Lattice nested Universe IDs.
 
         # 3D Lattices
-        if len(self._dimension) == 3:
-            for z in range(self._dimension[2]):
-                for y in range(self._dimension[1]):
-                    for x in range(self._dimension[0]):
-
-                        universe = self._universes[x][y][z]
-
-                        # Append Universe ID to the Lattice XML subelement
-                        universe_ids += '{0} '.format(universe._id)
-
-                        # Create XML subelement for this Universe
+        if self._num_axial is not None:
+            slices = []
+            for z in range(self._num_axial):
+                # Initialize the center universe.
+                universe = self._universes[z][-1][0]
+                universe.create_xml_subelement(xml_element)
+
+                # Initialize the remaining universes.
```

Comment:
Well thanks for pointing that out to me! I've been very slow to adopt PEP8 and am quickly realizing I should use it for everything from the start to satisfy the Pythonistas out there. Don't worry about it, clearly I should be the one adding periods to everything.